catalog-info.yaml

```diff
@@ -33,6 +33,11 @@ spec:
         build_pull_request_forks: false
         publish_commit_status: true
         build_branches: true
+      schedules:
+        Daily on main:
```

Comment:
We want to run DRA builds on a regular basis as well. First, because artifact retention is relatively short, and often times they are reaped before new commits are made, particularly old branches. Second, these builds use snapshot builds of Elasticsearch and we want to test that works periodically.

Reply:
How's it look now? 2x per day for each branch for both staging and snapshot?

Reply:
Yeah, that should be fine.

Reply:
Do we have a plan yet to update all these explicit branch declarations?

Reply:
I moved them to dynamic pipelines: https://github.com/elastic/elasticsearch-hadoop/pull/2114/files#diff-baef2c2baeae557c0a33867c9173c32b93e98beabd99bfece8d0f2df666566ab

They still have to be updated on version bump, but it's at least quicker / more straightforward

Reply:
Cool. Let's make sure we capture all these locations so we can make updating them part of our feature freeze process.